test/tracing/timetodisplaynative.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { getRNSentryOnDrawReporter } from "../../src/js/tracing/timetodisplaynative";
+import { ReactNativeLibraries } from "../../src/js/utils/rnlibraries";
+import type { ReactNativeLibrariesInterface } from "../../src/js/utils/rnlibrariesinterface";
+
+describe('timetodisplaynative', () => {
+  describe('on Web', () => {
+    jest.mock('../../src/js/utils/rnlibraries', () => {
+      const actual = jest.requireActual('../../src/js/utils/rnlibraries.web');
+      return {
+        ReactNativeLibraries: actual.ReactNativeLibraries as ReactNativeLibrariesInterface
+      };
+    });
+
+    test('requireNativeComponent to be defined', () => {
+      const test = ReactNativeLibraries.ReactNative.requireNativeComponent;
+      expect(test).toBeDefined();
```

Comment:
This test should check the behaviour on Web where requireNativeComponent is not defined. But it states it's defined.